hosting.md

```diff
@@ -1,6 +1,9 @@
-# Hosting a Bitcoin Core Review Club Meeting
+---
+layout: default
+title: Hosting a Review Club Meeting
+---
 
-Some tips for hosting a great review club meeting:
+# Hosting a Bitcoin Core Review Club Meeting
 
```

Comment:
For here or the future, a section on "how to choose a suitable PR" perhaps? And a note that it's totally fine to host a PR authored by yourself (one person asked me thinking that's not okay)

Reply:
I've had a first attempt at this. Let me know what you think.